lib/reporters/fps.js

```diff
@@ -1,9 +1,10 @@
 const babar = require('babar');
 
 function chart (data, url) {
-  return babar(data.map((fps, i) => {
+  var series = data.map((fps, i) => {
     return [i, fps];
-  }), { caption: `scrolling fps - ${url}`, height: 15 });
+  })
+  return babar(series, { caption: `scrolling fps - ${url}`, height: 12, maxX: 20, minY: 1, maxY: 60 });
```

Comment:
Also, we may need to review long-running fps experiments. as in fact we may not need any.
If something is bad for more than 20s, who cares? It's just bad, and there is no need to measure it further.